src/js/visual/configurator.js

```diff
@@ -276,6 +276,7 @@ ripe.Configurator.prototype.highlight = function(part, options) {
     var frontMask = this.element.querySelector(".front-mask");
     var src = frontMask.getAttribute("src");
     if (src === url) {
+        this.element.classList.add("highlight");

```

Comment:
@acaldas I think it's better to move the highlight operation (at the end of the function) up in the function (before this return) that way you only have one highlight add class operation